Movies/Models/MovieModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MovieModel.swift
+//  Movies
+//
+//  Created by Tyler Green on 5/12/23.
+//
+
+import Foundation
+
+struct SearchResults: Codable {
+    let results: [MovieModel]
+}
+
+struct MovieModel: Codable {
+    let id: Int
+    var title: String
```

Comment:
Is there a scenario you can think of where you would need to change the title of a movie?

Reply:
I guess not, are you saying it should be a `let` then?

Reply:
Yep if the property is not something that can change you will want to use `let`